src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideRunner.jsx

```diff
@@ -312,17 +315,17 @@ const PyodideRunner = (props) => {
     }
   };
 
-  if (!pyodideWorker) {
-    console.error("PyodideWorker is not initialized");
+  if (!pyodideWorker && active) {
+    console.warn("PyodideWorker is not initialized");
     return;
   }
 
   return (
     <div
-      className={`pythonrunner-container pyodiderunner${
-        active ? " pyodiderunner--active" : ""
-      }`}
-      style={{ display: showRunner ? "flex" : "none" }}
+      className={classNames("pythonrunner-container", "pyodiderunner", {
+        "pyodiderunner--active": active,
+      })}
+      style={{ display: active ? "flex" : "none" }}
```

Comment:
would it be better to use the `pyodiderunner--active` class we're defining, rather than doing this explicitly here?

Reply:
yep, that was a blind copy/paste but that's much nicer!